examples/mnist/.tensorleap/config.toml

```diff
@@ -1,7 +1,7 @@
 projectName = "mnist"
 organization = "dani"
 framework = "tensorflow"
-datasetName = "mnist"
+datasetName = "tmp"
```

Comment:
Are you sure?

Reply:
changed back (it was for adding two nets versions)